src/layouts/docs/docs-layout.module.css

```diff
@@ -25,31 +25,261 @@
   height: 100%;
   justify-content: center;
   overflow-y: auto;
-  padding: 24px 24px 100px 24px;
+  padding-bottom: 100px;
+  padding-left: 48px;
+  padding-right: 24px;
+  padding-top: 32px;
   position: relative;
 }
 
-.versionSwitcher {
-  height: 18px;
-  width: 346px;
-}
-
 /* TODO: these styles will be changing very soon, this is just to get us started */
 .main {
+  color: #3b3d45;
+  flex-grow: 1;
+  flex-shrink: 1;
+  font-size: 16px;
+  line-height: 22px;
   max-width: 804px;
   padding: 0 24px;
   padding-bottom: 50vh;
+  width: 0;
 
-  & h1:first-child {
-    margin-bottom: 12px;
+  & :global {
+    & a.__permalink-h {
+      display: none;
+    }
+
+    & .alert.alert-info {
+      background-color: #f2f8ff;
+      border: 1px solid #cce3f3;
+      border-radius: 6px;
+      color: #3b3d45;
+      padding: 24px;
+
+      & code {
+        background-color: white;
+        border: 1px solid #cce3fe;
+        border-radius: 5px;
+        color: inherit;
+      }
+
+      & p {
+        &:first-child {
+          margin-top: 0;
+        }
+
+        &:last-child {
+          margin-bottom: 0;
+        }
+      }
+    }
   }
 
+  & h1,
+  & h2,
+  & h3,
+  & h4,
+  & h5,
+  & h6 {
+    color: #0c0c0e;
+
+    & + ul,
+    & + ol {
+      margin-top: 12px;
+    }
+
+    & + p {
+      margin-top: 0;
+    }
+  }
+
+  & h1,
   & h2 {
     margin-bottom: 12px;
-    margin-top: 40px;
+  }
+
+  & h3,
+  & h4,
+  & h5,
+  & h6 {
+    margin-bottom: 8px;
+  }
+
+  & h4,
+  & h5,
+  & h6 {
+    color: #3b3d45;
+  }
+
+  /* named Heading 0 */
+  & h1 {
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 38px;
+    margin-top: 0;
+  }
+
+  /* named Heading 1 */
+  & h2 {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 32px;
+    margin-top: 48px;
+  }
+
+  /* named Heading 2 */
+  & h3 {
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 24px;
+    margin-top: 32px;
+  }
+
+  /* named Heading 3 */
+  & h4 {
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 24px;
+    margin-top: 24px;
+  }
+
+  /* named Heading 4 and Heading 5 */
+  & h5,
+  & h6 {
+    font-size: 13px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  & p {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    margin: 16px 0;
+  }
+
+  & strong {
+    font-weight: 500;
+  }
+
+  & ul,
+  & ol {
+    margin: 24px 0;
+    padding-left: 24px;
+
+    & li {
+      line-height: 24px;
+
+      &:not(:last-child) {
+        margin-bottom: 16px;
+      }
+
+      & ul,
+      & ol {
+        margin: 16px 0;
+      }
+    }
+  }
+
+  & a {
+    color: #1f69ff;
+    font-weight: 500;
+    text-decoration: underline;
+
+    &:visited {
+      color: #a737ff;
+    }
   }
 
   & img {
     max-width: 100%;
   }
+
+  /* inline code elements */
+  & code {
```

Comment:
In [our `react-content` styles](https://github.com/hashicorp/react-components/blob/1b3791abfef10b96b05495598ce313524be9213e/packages/content/styles/code-inline.module.css#L3) we do ~ `& :not(pre) code` to avoid these styles bleeding into our swapped-in-via-MDX code block. Not sure if these are necessary, but would recommend as it can act as a light guarantee we won't clobber custom styles (and may also remove the need for the `& pre code` "reset"-ish styles below).

Reply:
This is a great point, I'll make this change!

Reply:
I wasn't successful in making this change, and since we'll be styling code blocks differently at some point I'm not super concerned for now. We can come back to this if needed!